forge-gui/res/cardsfolder/upcoming/atraxas_skitterfang.txt

```diff
@@ -0,0 +1,9 @@
+Name:Atraxa's Skitterfang
+ManaCost:3
+Types:Artifact Creature Phyrexian Insect
+PT:2/2
+K:etbCounter:OIL:3
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | OptionalDecider$ You | Execute$ TrigPump
+SVar:TrigPump:AB$ Pump | Cost$ SubCounter<1/OIL> | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select a target creature you control | KWChoice$ Flying,Vigilance,Deathtouch,Lifelink | StackDescription$ SpellDescription | SpellDescription$ At the beginning of combat on your turn, you may remove an oil counter from CARDNAME. When you do, target creature you control gains your choice of flying, vigilance, deathtouch, or lifelink until end of turn.
```

Comment:
- TriggerDescription missing instead of unused Stack + SpellDescription
- reflexive trigger not handled again